### DISCUSSION TITLE
Model display black silhouette

### DISCUSSION THREAD

**Discussion by AdrianTabacaru** [google/model-viewer#3329] [General]
Hello everyone, I see there are different posts about this but couldn't figure out how to fix it so I'm posting it again hoping for a updated answer.
I'm testing a page on different devices and the model shows as black silhouette on two devices:
1. iPhone with iOS 14.4
2. macOS Catalina - 10.15.7 with Chrome v.99
3. on Safari v.13.1.3 the page is totally white.
Can you please help me get this working on all?

On other devices such as iPhone iOS 15.4 or on windows with Chrome there are no issues and everything displays correctly.

Thank you!
Adrian

![macOS Catalina- 10 15 7](https://user-images.githubusercontent.com/26578826/161002607-6970f96d-256b-4f28-9805-ef35863ffa6d.png)
![c5264ae3-85de-451d-8eaf-f7fea7419af4](https://user-images.githubusercontent.com/26578826/161002612-62e6fe3c-9cc5-4bc8-85bd-29ecb0ad90cf.jpg)

**Comment by elalish** [comment]
Please include a link to a site where we can repro or at least the GLB. Otherwise we have no way to debug. Also, is the result different with different MV versions?

**Comment by AdrianTabacaru** [comment]
Hello, thank you for the answer.
The page is here:
https://adrian-tabacaru.com/WEB_AR/CutieDeseuriBiologice-10Lv2/WebAR_cutie_deseuri_10lv2.html
I think…I use the latest version of mv. 
Thank You!

**Comment by elalish** [comment]
Thanks; we generally only officially test and support the last two releases of browsers. Especially with Safari, as older versions tend to have a lot of WebGL bugs. Our automated tests do cover iOS 14, so we're not getting black renders there. Can you check if any prior model-viewer versions worked? e.g. https://unpkg.com/@google/model-viewer@1.8.0/dist/model-viewer.js
